_format/1.1/index.md

```diff
@@ -2226,6 +2226,28 @@ because the value for the `updated` key is not recognized under the profile's
 requirement that the `updated` key hold a string of the form produced by 
 `JSON.stringify`.
 
+### <a href="#profiles-required" id="profiles-required" class="headerlink"></a> Server-Required Profiles 
+
+Generally, servers should not require that clients apply specific profiles to 
+their requests, as such requirements reduce interoperability.
+
+However, in many common cases, a server does require that clients provide 
+certain data (e.g., authentication credentials), and it **MAY** require that 
+this data be provided by applying a specific profile (or any one profile from 
```

Comment:
I find this example not very convincing. I cannot imagine authentication is often embedded into JSON:API documents. Authentication is an orthogonal concern.

Which leaves me wondering: what _is_ a good example of this? Why is this "common"?

Reply:
> A server **MAY** require that a client applies a specific profile to its request.
>
> However, it is **NOT RECOMMENDED** for servers to require profiles which require additional query parameters or additional request document members, as such requirements reduce interoperability between clients and servers.
>
> >Note: Profiles with optional document members, specific value formats (e.g. ISO 8601 date and time format), or specific query parameters within a query parameter family already reserved by this specification should be treated with care, but do not rise to the same level interoperability concerns.

Reply:
My theory here is that:

1. anything optional here obviously doesn't have interop concerns.
2. query parameters within query parameter families are usually implementation specific (filter, sort, etc) so clients will likely not have them hardcoded.
3. like #2, value formats are usually "passed through" a client library and aren't hardcoded; they typically come from a frontend developer working against a specific backend.